bluesky_queueserver/profile_collection_sim/user_group_permissions.yaml

```diff
@@ -3,11 +3,11 @@ user_groups:
     allowed_plans:
       - null  # Allow all
     forbidden_plans:
-      - "$_"  # All plans with names starting with '_'
+      - "^_"  # All plans with names starting with '_'
```

Comment:
I guess the old version never worked, as it was looking for an impossible combination of the end of the line and `_`?

Reply:
I assume it was never used